src/osd/ReplicatedBackend.cc

```diff
@@ -653,13 +653,15 @@ int ReplicatedBackend::be_deep_scrub(
       pos.data_hash = bufferhash(-1);
     }
 
+    uint64_t stride = cct->_conf->osd_deep_scrub_stride;
```

Comment:
As a style suggestion:
const utint6_t ....

Reply:
> As a style suggestion: const utint6_t ....

OK, I have modified it. Thanks!